room/compiler-processing/src/main/java/androidx/room/compiler/processing/javac/JavacProcessingEnv.kt

```diff
@@ -73,6 +73,16 @@ internal class JavacProcessingEnv(
         return typeElementStore[qName]
     }
 
+    override fun getTypeElementsFromPackage(packageName: String): List<XTypeElement> {
```

Comment:
FYI i just merged support for top level members so we can support them as a followup to this.